aict_tools/configuration.py

```diff
@@ -156,12 +155,17 @@ class DispConfig:
         'delta_column',
         'log_target',
         'project_disp',
+        'coordinate_transformation',
     ]
 
     def __init__(self, config):
         model_config = config['disp']
 
         self.coordinate_transformation = model_config['coordinate_transformation']
+        if self.coordinate_transformation not in ['CTA', 'FACT']:
+            raise Exception(
+                '''Value of coordinate_transformation needs to be set
```

Comment:
Why three quotes?

Reply:
Wasnt sure about line length constrains.

Reply:
You can use single quotes like this

```
raise Exception(
     'Value of coordinate_transformation needs to be set'
     'to bla bla'
)
```

Using the triple-quotes will result in the indentation and the linebreak being in the error message, which is not wanted here I think

Reply:
```
In [1]: msg = '''Hello World 
   ...:       foo'''                                                                                                                          

In [2]: print(msg)                                                                                                                            
Hello World
      foo
```

Reply:
Ah, I thought that needs extra brackets.
But 80 character limit is not enforced anyway, right?

Reply:
No. But it should not exceed 90